### ISSUE TITLE
Support tiling

### ISSUE THREAD

**Issue by gartia** [leejet/stable-diffusion.cpp#106]
I made a [pr to ggml](https://github.com/ggerganov/ggml/pull/638) to add circular padding, would SD need anything else for this?

It seems like[ pytorch SD](https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/4afaaf8a020c1df457bcf7250cb1c7f609699fa7/modules/sd_hijack.py#L339) implementation you just replace the 0 padding with a circular method. 
```py

    def apply_circular(self, enable):
        if self.circular_enabled == enable:
            return

        self.circular_enabled = enable

        for layer in [layer for layer in self.layers if type(layer) == torch.nn.Conv2d]:
            layer.padding_mode = 'circular' if enable else 'zeros'```

Is 0 padding already implemented in SD cpp? if so would it also just be this easy 

There is also more advanced methods such as torus wrapping the tensor, but i would like to just test this before something like that
